test2code-agent/src/main/kotlin/com/epam/drill/plugins/test2code/checksum/Checksum.kt

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright 2020 - 2022 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.epam.drill.plugins.test2code.checksum
+
+import com.epam.drill.plugins.test2code.common.api.AstEntity
+import com.epam.drill.plugins.test2code.common.api.AstMethod
+import org.apache.bcel.classfile.ClassParser
+import org.apache.bcel.classfile.Method
+import org.jacoco.core.internal.data.CRC64
+import java.io.ByteArrayInputStream
+
+internal fun checksumCalculation(
+    classBytes: ByteArray,
+    className: String,
+    astClass: AstEntity
```

Comment:
pass `List<AstMethod>` instead of the whole `AstEntity`

Reply:
@bodyangug

Reply:
done

Reply:
done